public/electron.js

```diff
@@ -195,11 +196,30 @@ ipc.on('data', (event, args) => {
     //write the data
     stream.write(JSON.stringify({...args, git}))
 
-    // Copy provocation images to patient's data folder
+    // Copy provocation images to participant's data folder
     if (args.trial_type === 'image_keyboard_response') images.push(args.stimulus.slice(7))
   }
 })
 
+// Save Video
+ipc.on('save_video', (event, fileName, buffer) => {
+  
+  const desktop = app.getPath('desktop')
+  const name = app.getName()
+  const today = new Date(Date.now())
+  const date = today.toISOString().slice(0,10)
+  const fullPath = path.join(desktop, dataDir, studyID, participantID, date, name, fileName)
```

Comment:
this path doesn't match the general data saving path, but they probably should.  This one is more specific so maybe go with this one?

Reply:
or maybe the one below is the correct place and this can be ignored

Reply:
The path for the video and the data seems the same to me.